conda_rpms/install.py

```diff
@@ -40,13 +41,17 @@
 import json
 import shutil
 import stat
+from stat import S_IMODE, S_IXGRP, S_IXOTH, S_IXUSR
 import sys
 import subprocess
 import tarfile
+from textwrap import dedent
 import traceback
 import logging
+import re
 import shlex
-from os.path import abspath, basename, dirname, isdir, isfile, islink, join
+from os.path import abspath, basename, dirname, exists, isdir, isfile, islink, \
+    join, split, splitext
```

Comment:
@lbdreyer Do we care about sorting the import order here?

Reply:
Not sure. It is a copy of the conda/install.py and I wanted to make minimal changes, but I also wanted to have minimal code necessary (rather than it be some long thing).
But I think we could reorder the imports, as long as we put in a comment saying that we did that.